ShotHandler/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/ShotHandlerFieldStatusEventTest.java

```diff
@@ -0,0 +1,35 @@
+package pl.jakubowskiprzemyslaw.tajgertim;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pl.jakubowskiprzemyslaw.tajgertim.event.FieldStatusEvent;
+import pl.jakubowskiprzemyslaw.tajgertim.models.coordinates.FieldStatus;
+import pl.jakubowskiprzemyslaw.tajgertim.service.ShotHandlerQueueListener;
+
+import static org.mockito.Mockito.mock;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+
+@Test
+public class ShotHandlerFieldStatusEventTest {
+
+    private FieldStatusEvent event;
+    private FieldStatus status;
+
+    @BeforeMethod
+    public void setUp() {
+        ShotHandlerQueueListener queueListener = mock(ShotHandlerQueueListener.class);
```

Comment:
try to use MockitoAnnotations.initMocks(this); when you initialize many mocks. 
Remember about add @Mock annotation